nextcord/application_command.py

```diff
@@ -809,7 +809,7 @@ def __init__(
         super().__init__(callback=callback, cmd_type=cmd_type, name=name, description=description)
         self._state: Optional[ConnectionState] = None
         self.force_global: bool = force_global
-        self.default_permission: bool = default_permission or True
+        self.default_permission: Optional[bool] = default_permission if default_permission is not None else True
```

Comment:
Suggestion

See https://github.com/hlieberman/nextcord/blob/f94b7b44f8a43f948e833dae19b08f365b81de05/nextcord/application_command.py#L980-L981.

If default_permissions is None, it will use Discord's default, so this should not default to True at all.

```diff
- self.default_permission: Optional[bool] = default_permission if default_permission is not None else True
+ self.default_permission: Optional[bool] = default_permission
```